docs/backup/10-01-restore-troubleshooting.md

```diff
@@ -33,3 +33,23 @@ If one or more channels report as not ready, delete their corresponding channel
 kubectl delete service -l messaging.knative.dev/role=natss-channel -n kyma-system
 kubectl annotate natsschannels.messaging.knative.dev -n kyma-system restore=done --all
 ```
+
+## AssetGroup restoration fails due to duplicated default buckets
+
+Velero restores Kyma resources alphabetically, following the `kind` object order. This creates issues for Rafter as Valero first restores (Cluster)AssetGroups and only then the default (Cluster)Buckets. As the (Cluster)AssetGroup controller automatically tries to create the default buckets that Velero restored in the meantime, (Cluster)AssetGroups restoration will fail due to too many buckets with the `rafter.kyma-project.io/access:public` label.
```

Comment:
As the (Cluster)AssetGroup controller automatically tries to create the default buckets when default bucket is not available...